swagger-spec/registrations/definition.yaml

```diff
@@ -141,6 +144,19 @@ properties:
         type: string
         readOnly: true
         description: 'The reasoning for why this registration was withdrawn, as entered by the administrator that initiated the withdrawal.'
+      draft_registration:
+        type: string
+        readOnly: false
+        description: 'The unique ID of the draft registration from which the registration was created.'
+      registration_choice:
+        type: string
+        readOnly: false
+        description: 'Type of registration you want to choose to be, immediate or embargo. If you choose to be embargo, you need to provide with a lift_embargo datetime.'
```

Comment:
Rewording: just a suggestion:
Describes when the registration will become public, either "immediate" or "embargo".  If this field is "embargo", you will need to supply a datetime in the "lift_embargo" field.